Cargo.lock

```diff
@@ -18,13 +18,34 @@ version = "0.1.5"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "a13739d7177fbd22bb0ed28badfff9f372f8bef46c863db4e1c6248f6b223b6e"
 
+[[package]]
```

Comment:
This file is till commited. Delete it from the repo

Reply:
Done